src/components/dialogmote/innkalling/BehandlerSearch.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useRef, useState } from "react";
+import "@navikt/ds-css";
+import { Search } from "@navikt/ds-react";
+import BehandlerSearchResult from "@/components/dialogmote/innkalling/BehandlerSearchResult";
+
+const BehandlerSearch = (): ReactElement => {
+  const searchRef = useRef(null);
```

Comment:
Hva brukes denne til, siden den bare settes til `null`?

Reply:
Den brukes i `Popover` i `BehandlerSearchResult` etter forslag fra Aksel: https://aksel.nav.no/designsystem/komponenter/search?#demo (velg Komposisjon "med søketreff").
Er ikke så kjent med useRef utenfor det, men mistenker at det er for knytte søkefeltet opp mot popoveren. Om dette er nødvendig kan vi jo undersøke mer 🤔 
https://dev.to/nibble/what-is-useref-hook-and-how-do-you-use-it-5emo